tests/net/socket/getaddrinfo/src/main.c

```diff
@@ -47,7 +259,9 @@ void test_main(void)
 	k_thread_system_pool_assign(k_current_get());
 
 	ztest_test_suite(socket_getaddrinfo,
-			 ztest_user_unit_test(test_getaddrinfo_ok),
+			 ztest_unit_test(test_getaddrinfo_setup),
+			 ztest_unit_test(test_getaddrinfo_ok),
```

Comment:
why is test_getaddrinfo_ok no longer running in user mode?
all it does is call getaddrinfo() and take a semaphore.

Reply:
I will convert it to use sys_mutex so that the test case works in user mode.